ssl/ssl_asn1.c

```diff
@@ -37,25 +37,25 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include "ssl_locl.h"
-#include <openssl/asn1t.h>
+#include "internal/asn1t.h"
 #include <openssl/x509.h>
 
 typedef struct {
-    long version;
-    long ssl_version;
+    int32_t version;
```

Comment:
I tried again, and it works like a charm...  Maybe I confused it with something else...